### PR TITLE
Update DeveloperGuide.md

### PR DESCRIPTION
**Pull request by Irminrics**
This commit
- changes Delete a Person to Delete an Event under Appendix: Instructions for manual testing in DeveloperGuide.md.
- adds use case for `findevent` and instruction for manual testing for `listall`, `listevcontact` and `findevent`. 
- change all existing contacts to persons

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -512,32 +512,32 @@ testers are expected to do more *exploratory* testing.
...
 
-   4. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+   4. Other incorrect delete commands to try: `delevent`, `delevent x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-2. Deleting a person while persons with a specified keyword, i.e., `[KEYWORD]` of the user's choice, are being shown
+2. Deleting an event while event with a specified keyword, i.e., `[KEYWORD]` of the user's choice, are being shown
 
-    1. Prerequisites: List all persons using the `list` command, then find all persons with `[KEYWORD]` using the `find [KEYWORD]` command. Multiple persons in the list.
+    1. Prerequisites: Find all events with `[KEYWORD]` using the `find [KEYWORD]` command. Multiple events in the list.
```
Comment (inas0ng) [inline docs/DeveloperGuide.md:532]:
```suggestion
    1. Prerequisites: Find all events with `[KEYWORD]` using the `findevent [KEYWORD]` command. Multiple events in the list.
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -512,32 +512,32 @@ testers are expected to do more *exploratory* testing.
...
-      Expected: No person is deleted. Error details shown in the status message.
+   3. Test case: `delevent 0`<br>
+      Expected: No event is deleted. Error details shown in the status message.
 
-   4. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
+   4. Other incorrect delete commands to try: `delevent`, `delevent x`, `...` (where x is larger than the list size)<br>
       Expected: Similar to previous.
 
-2. Deleting a person while persons with a specified keyword, i.e., `[KEYWORD]` of the user's choice, are being shown
+2. Deleting an event while event with a specified keyword, i.e., `[KEYWORD]` of the user's choice, are being shown
```
Comment (inas0ng) [inline docs/DeveloperGuide.md:530]:
```suggestion
2. Deleting an event while events with a specified keyword, i.e., `[KEYWORD]` of the user's choice, are being shown
```

**Review by inas0ng — COMMENTED** [review]
just a few small typos to fix!

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -407,6 +407,25 @@ Use case ends.
 
 <br>
 
+**Use case: Find an event**
+
+**MSS**
+
+1.  User requests to find an event based on a given keyword.
+2.  PlanEase  shows the list of sorted events based on the user’s choice of keyword.
```
Comment (venuslimm) [inline docs/DeveloperGuide.md:415]:
```suggestion
2.  PlanEase shows the list of sorted events based on the user’s choice of keyword.
```

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -512,34 +531,57 @@ testers are expected to do more *exploratory* testing.
 4. Other incorrect addevent commands to try: `addevent`, `...`<br>
    Expected: Similar to previous.
 
-### Deleting a person
+### Finding an event
+
+1. Prerequisite: Current list of events has only one event with the word `Carnival` and no event with the word `Magic`.
```
Comment (venuslimm) [inline docs/DeveloperGuide.md:536]:
```suggestion
1. Prerequisites: Current list of events has only one event with the word `Carnival` and no event with the word `Magic`.
```

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -512,34 +531,57 @@ testers are expected to do more *exploratory* testing.
...
-   1. Prerequisites: List all persons using the `list` command. Multiple persons in the list.
-   
-   2. Test case: `delete 1`<br>
-      Expected: First contact is deleted from the list. Details of the deleted contact shown in the status message.
+3. Test case: `findevent Magic`<br>
+   Expected: No matching event is found.
 
-   3. Test case: `delete 0`<br>
-      Expected: No person is deleted. Error details shown in the status message.
+4. Other incorrect findevent commands to try: `findevent`, `...`<br>
```
Comment (venuslimm) [inline docs/DeveloperGuide.md:544]:
```suggestion
4. Other incorrect findevent commands to try: `findevent`, `...`.<br>
```

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -512,34 +531,57 @@ testers are expected to do more *exploratory* testing.
...
+
+   1. Prerequisites: List all events using the `listevent` command. Multiple events in the list.
+
+   2. Test case: `delevent 1`<br>
+      Expected: First event is deleted from the list. Details of the deleted event shown in the status message.
+
+   3. Test case: `delevent 0`<br>
+      Expected: No event is deleted. Error details shown in the status message.
 
    4. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
```
Comment (venuslimm) [inline docs/DeveloperGuide.md:559]:
```suggestion
   4. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size).<br>
```

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -512,34 +531,57 @@ testers are expected to do more *exploratory* testing.
 4. Other incorrect addevent commands to try: `addevent`, `...`<br>
```
Comment (venuslimm) [inline docs/DeveloperGuide.md:531]:
```suggestion
4. Other incorrect addevent commands to try: `addevent`, `...`.<br>
```

**Review by venuslimm — COMMENTED** [review]
I've left some suggestions but other than those, looks good to me!

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -509,37 +528,60 @@ testers are expected to do more *exploratory* testing.
...
+    2. Test case: `delevent 1`<br>
+       Expected: First event is deleted from the displayed list. Details of the deleted event shown in the status message.
 
-    3. Test case: `delete 0`<br>
-       Expected: No person is deleted. Error details shown in the status message.
+    3. Test case: `delevent 0`<br>
+       Expected: No event is deleted. Error details shown in the status message.
 
-    4. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the displayed list size)<br>
+    4. Other incorrect delete commands to try: `delevent`, `delevent x`, `...` (where x is larger than the displayed list size)<br>
```
Comment (venuslimm) [inline docs/DeveloperGuide.md:572]:
```suggestion
    4. Other incorrect delete commands to try: `delevent`, `delevent x`, `...` (where x is larger than the displayed list size).<br>
```

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -509,37 +528,60 @@ testers are expected to do more *exploratory* testing.
...
+      Expected: First event is deleted from the list. Details of the deleted event shown in the status message.
+
+   3. Test case: `delevent 0`<br>
+      Expected: No event is deleted. Error details shown in the status message.
+
+   4. Other incorrect delete commands to try: `delevent`, `delevent x`, `...` (where x is larger than the list size).<br>
       Expected: Similar to previous.
 
-2. Deleting a person while persons with a specified keyword, i.e., `[KEYWORD]` of the user's choice, are being shown
+2. Deleting an event while events with a specified keyword, i.e., `[KEYWORD]` of the user's choice, are being shown
```
Comment (inas0ng) [inline docs/DeveloperGuide.md:562]:
```suggestion
2. Deleting an event while events with a specified keyword, i.e., `KEYWORD` of the user's choice, are being shown
```

File: docs/DeveloperGuide.md [hunk 10]
```diff
@@ -509,37 +528,60 @@ testers are expected to do more *exploratory* testing.
...
 
-    1. Prerequisites: List all persons using the `list` command, then find all persons with `[KEYWORD]` using the `find [KEYWORD]` command. Multiple persons in the list.
+    1. Prerequisites: Find all events with `[KEYWORD]` using the `findevent [KEYWORD]` command. Multiple events in the list.
```
Comment (inas0ng) [inline docs/DeveloperGuide.md:564]:
```suggestion
    1. Prerequisites: Find all events with `KEYWORD` using the `findevent KEYWORD [MORE_KEYWORDS]` command. Multiple events in the list.
```

**Review by inas0ng — COMMENTED** [review]
sorry adding 2 more changes to be consistent with the other write-ups!

File: docs/DeveloperGuide.md [hunk 11]
```diff
@@ -407,6 +407,25 @@ Use case ends.
 
 <br>
 
+**Use case: Find an event**
+
+**MSS**
+
+1.  User requests to find an event based on a given keyword.
+2.  PlanEase shows the list of sorted events based on the user’s choice of keyword.
```
Comment (weilin1202) [inline docs/DeveloperGuide.md:415]: I think 'filtered' would be a more suitable word as compared to 'sorted'

**Review by jerome-neo — APPROVED** [review]
LGTM!

**Pull request merged**